specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/IntegrationRuntime.json

```diff
@@ -1011,6 +1011,223 @@
         "name": "SsisObjectMetadataType",
         "modelAsString": true
       }
+    },
+    "SsisFolder": {
```

Comment:
The changes also need to be in version 2017-09-01-preview.

Reply:
@zhangyd2015  The changes in private swagger not betting regenerated into SDKs, so no need to change there.